vcd/datasource_vcd_org_vdc_test.go

```diff
@@ -69,14 +70,13 @@ func TestAccVcdVdcDatasource(t *testing.T) {
 			return
 		}
 
-		validateResourceAndDataSource(t, configText, datasourceVdc)
+		validateResourceAndDataSource(t, configText, datasourceVdc, params)
 	}
 	postTestChecks(t)
 }
 
-func validateResourceAndDataSource(t *testing.T, configText string, datasourceVdc string) {
+func validateResourceAndDataSource(t *testing.T, configText string, datasourceVdc string, params StringMap) {
```

Comment:
Maybe I lost some detail but `params` is not used inside this function, is it?

Reply:
yes, removed